<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>778</width>
      <height>578</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 171, 172);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="DarkShade">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>778</width>
       <height>578</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,50);
border-radius:15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_button">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Stencil</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#exit_button{
	background-color:rgb(184, 67, 44);
	border-radius:5px;
color:rgb(170, 170, 170);
}
QPushButton#exit_button:hover{	
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(136, 48, 33, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}
QPushButton#exit_button:pressed{
	padding-left:5px;
padding-top:5px;
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(4, 29, 36, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}


</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="Program_name">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>-10</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Dubai Medium</family>
       <pointsize>32</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>NoAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(22, 60, 73);
background:none;</string>
     </property>
     <property name="text">
      <string>Assistants</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#BackButton{
	
	
	background-color: rgb(11, 72, 89);
	border-radius:5px;
color:rgb(200,200,200);
}
QPushButton#BackButton:hover{	
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(19, 122, 150, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}
QPushButton#BackButton:pressed{
	padding-left:5px;
padding-top:5px;
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(4, 29, 36, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}

</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_Button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>170</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#delete_Button{
	
	
	background-color: rgb(11, 72, 89);
	border-radius:5px;
color:rgb(200,200,200);
}
QPushButton#delete_Button:hover{	
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(19, 122, 150, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}
QPushButton#delete_Button:pressed{
	padding-left:5px;
padding-top:5px;
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(4, 29, 36, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}

</string>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="first_Name">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>220</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255, 255, 255,230);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#add_Button{
	
	
	background-color: rgb(11, 72, 89);
	border-radius:5px;
color:rgb(200,200,200);
}
QPushButton#add_Button:hover{	
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(19, 122, 150, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}
QPushButton#add_Button:pressed{
	padding-left:5px;
padding-top:5px;
	background-color: qlineargradient(spread:pad, x1:0.99, y1:1, x2:0, y2:0, stop:0.119403 rgba(4, 29, 36, 255), stop:0.791045 rgba(37, 44, 38, 255));
	border-radius:5px;
color:rgb(170, 170, 170);
}

</string>
     </property>
     <property name="text">
      <string>add assistant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="salary">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>220</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255, 255, 255,230);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>salary</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="last_Name">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>220</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255, 255, 255,230);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="teacher_Id_Box">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255, 255, 255,230);
padding-bottom:7px;
font: 57 12pt &quot;Dubai Medium&quot;;</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>choose a teacher</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="note_msg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(143,0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="assistant_box">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 12pt &quot;Dubai Medium&quot;;
background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255, 255, 255,230);
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>choose an assistant</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="gender">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255, 255, 255,230);
padding-bottom:7px;
font: 57 12pt &quot;Dubai Medium&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>choose a gender</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>first_Name</tabstop>
  <tabstop>last_Name</tabstop>
  <tabstop>salary</tabstop>
  <tabstop>gender</tabstop>
  <tabstop>teacher_Id_Box</tabstop>
  <tabstop>add_Button</tabstop>
  <tabstop>assistant_box</tabstop>
  <tabstop>delete_Button</tabstop>
  <tabstop>BackButton</tabstop>
  <tabstop>exit_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
